test/catalog/operatorhubio/shipwright-operator-bundle.json

```diff
@@ -20,6 +20,14 @@
                 "version": "v1alpha1"
             }
         },
+        {
+            "type": "olm.gvk.required",
+            "value": {
+                "group": "cert-manager.io",
+                "kind": "Certificate",
+                "version": "v1"
+            }
+        },
```

Comment:
Noticing here - our test is triggering OLM dependency resolution (see https://olm.operatorframework.io/docs/concepts/olm-architecture/dependency-resolution/#declaring-dependencies), as it is also doing with Tekton.

Does the Shipwright operator today not do automatic dependency resolution?

Reply:
I ve tested two ways to declare dependencies for the operator:
- Adding a Required section to the csv with the required APIs: olm installs automatically the dependencies as they are part of the subscription
- Adding dependencies.yaml file to metadata, olm blocks operator installation until the dependencies are manually installd by an admin
I am keeping the required Section and not pushing the dependencies.yaml file for now